### ISSUE TITLE
GUI Test HandsOn4 does not run on travis.

### ISSUE THREAD

**Issue by PaulWestphal** [FeatureIDE/FeatureIDE#891]
The test runs fine for me locally, on the develop branch with Eclipse Proton and Eclipse Oxygen. I've tried adding waits, try blocks that close the eclipse error reporting popup in case it appears, and changed the way that DirectedCall is typed. I've even re-recorded the entire GUI test (on the branch issue_791), but in the end I always get this error message from travis:

```
Failed to set selection:
[[Warnings \(4 items\), This annotation causes a dead code block.
The presence condition of Sabbath is a contradiction because:
• Sabbath and FIFO are alternatives \(i.e., ¬\(Sabbath ∧ FIFO\)\).
• Sabbath and ShortestPath are alternatives \(i.e., ¬\(Sabbath ∧ ShortestPath\)\).
• DirectedCall and UndirectedCall are alternative children of CallButtons \(i.e., CallButtons ⇒ DirectedCall ∨ UndirectedCall\).
• DirectedCall ⇒ ShortestPath is a constraint.
• UndirectedCall ⇒ FIFO ∨ ShortestPath is a constraint.
• The expression is nested within a block annotated with CallButtons.]].
```

**Comment by guimatsia** [comment]
Dear first last,

Thank you for signing up for an  O'Reilly account.

You can update your email address, password, and account preferences at any time by visiting:
    https://click.pstmrk.it/2tsm/www.oreilly.com%2Fmember%2Fprofile%2F./D6t4GQQ/1dwx/sVl3GS6dhw/V0VMQ09NRQ

Have questions? Email {{support_email}.

Regards,
The O'Reilly Team

O'Reilly Media, Inc.

1005 Gravenstein Hwy North

Sebastopol, CA 95472

© 2019 O'Reilly. All rights reserved.

**Comment by guimatsia** [comment]
@media only screen and (max-width:655px) {
    .fullwidth {
      width: 100% !important;
    }
  }

  @media only screen and (max-width:475px) {
    .mobile {
      width: 100% !important;
      text-align: left !important;
      padding-top: 15px !important;
    }
  }
  



div.preheader 
{ display: none !important; } 

Make the most of your free trial.
  
  
    
      
        
        
          
             
          
          
            
              
              
                
                  
                    
                    
                      
                        
                        
                      
                    
                    
                    
                      
                        
                          Online Learning • Conferences • Ideas
                        
                      
                    
                  
                
                
                   
                
                
                
                  
                    
                      
                        
                          
                          
                            
                              
                                Let's start learning.
                              
                            
                          
                          
                        
                      
                    
                  
                
                
                   
                
              
              
              

                
                
                  
                    
                    

                    first, welcome to O'Reilly online learning. For the next 10 days, you have unlimited online and offline access to fresh and relevant content from us and 200+ partners that will keep you ahead of what's next and help you advance your career. 

                    While your trial expires in 10 days, you can stay on the inside track with exclusive content and expert training by becoming a paid member.

                    Don't just take our word for it. Watch this video to hear from customers like you who are succeeding with O'Reilly.

                    Ready to get started?

                  
                

                
                
                    
                
                
                  
                    
                      
                        
                          
                            
                              Sign in
                            
                          
                        
                      
                    
                  
                
                

                 
                   
                
              
              
              
                
                   
                
                
                
                  
                    
                      
                        
                        
                        
                        
                        
                        
                        
                        
                        
                        
                      
                    
                  
                
                
                   
                
                
                
                
                  
                    This message was sent to reply@reply.github.com. You are receiving this because you're an O'Reilly online learning member, or you've signed up to receive email from us. We hope you found this message to be useful. However, if you'd rather not receive future emails of this type from O'Reilly, please manage your preferences or unsubscribe here.
                    
                    Read our Privacy Policy.
                    
                    O'Reilly Media, Inc., 1005 Gravenstein Highway North, Sebastopol, CA, 95472 (707) 827-7000

**Comment by guimatsia** [comment]
@media only screen and (max-width:655px) {
    .fullwidth {
      width: 100% !important;
    }
  }

  @media only screen and (max-width:475px) {
    .mobile {
      width: 100% !important;
      text-align: left !important;
      padding-top: 15px !important;
    }
  }
  



div.preheader 
{ display: none !important; } 

Our search tool makes it easy.
  
  
    
      
        
        
          
             
          
          
            
              
              
                
                  
                    
                    
                      
                        
                        
                      
                    
                    
                    
                      
                        
                          Online Learning • Conferences • Ideas
                        
                      
                    
                  
                
                
                   
                
                
                
                  
                    
                      
                        
                          
                          
                            
                              
                                The right content, right at your fingertips.
                              
                            
                          
                          
                        
                      
                    
                  
                
                
                   
                
              
              
              
                
                
                  
                    
                    first, when you have to solve a problem quickly, the idea of sorting through thousands of pieces of content can be overwhelming. 
                    That's where our powerful search comes in. Simply type what you're looking for into the search bar at the top of our homepage and get to the exact content you need, right away. Or, go broader and easily filter topics to narrow your focus.
                  
                
                
                
                    
                
                
                  
                    
                      
                        
                          
                            
                              Start searching now
                            
                          
                        
                      
                    
                  
                
                
                
                   
                
                
                
                  
                    
                      
                        
                          
                            
                              
                                Stay ahead of the curve.
                                Your trial is ending soon, but you don't have to lose access to O'Reilly online learning. Purchase a month-to-month membership for just $39 per month or a yearlong membership for just $399 (a 15% savings).
                                
                                  
                                    Become a member today
                                    
                                  
                                
                              
                            
                          
                        
                      
                    
                  
                
                
                   
                
              
              
              
                
                   
                
                
                
                  
                    
                      
                        
                        
                        
                        
                        
                        
                        
                        
                        
                        
                      
                    
                  
                
                
                   
                
                
                
                
                  
                    This message was sent to reply@reply.github.com. You are receiving this because you're an O'Reilly online learning member, or you've signed up to receive email from us. We hope you found this message to be useful. However, if you'd rather not receive future emails of this type from O'Reilly, please manage your preferences or unsubscribe here.
                    
                    Read our Privacy Policy.
                    
                    O'Reilly Media, Inc., 1005 Gravenstein Highway North, Sebastopol, CA, 95472 (707) 827-7000

**Comment by guimatsia** [comment]
@media only screen and (max-width:655px) {
    .fullwidth {
      width: 100% !important;
    }
  }

  @media only screen and (max-width:470px) {
    .mobile {
      width: 100% !important;
      text-align: left !important;
      padding-top: 15px !important;
    }


    .full-image {
      display: block;
      width: 100%;
      padding: 0 0 20px 0 !important;

    }

    .full-text {
      display: block;
      width: 100%;
      padding: 0 !important;

    }

  }
  



div.preheader 
{ display: none !important; } 

Content that caters to you
  
  
    
      
        
        
          
             
          
          
            
              
              
                
                  
                    
                    
                      
                        
                        
                      
                    
                    
                    
                      
                        
                          Online Learning • Conferences • Ideas
                        
                      
                    
                  
                
                
                   
                
                
                
                  
                    
                      
                        
                          
                          
                            
                              
                                Learning when you need it—and how you need it.
                              
                            
                          
                          
                        
                      
                    
                  
                
                
                   
                
              
              
              
                
                
                  
                    
                    first, 65% of our subscribers report that our on–demand content saves them an average of 1.2 hours per week—that's almost 8 days per year. Here's how.
                    
                      
                        
                          
                            
                              
                              
                              
                                Popular videos
                                Dive into the most pressing topics and emerging trends.
                                
                                  
                                    Check them out
                                    
                                  
                                
                              
                            
                          
                        
                      
                      
                          
                      
                      
                         
                      
                      
                        
                          
                            
                              
                              
                              
                                Learning Paths
                                Build skills sequentially with these immersive, expert–led experiences.
                                
                                  
                                    Get started
                                    
                                  
                                
                              
                            
                          
                        
                      
                      
                          
                      
                      
                         
                      
                      
                        
                          
                            
                              
                              
                              
                                O'Reilly app
                                Pressed for time? Watch a Case Study (conference session) during your commute.
                                
                              
                                
                                  
                                     
                                    
                                  
                                
                              
                            
                          
                        
                      
                      
                          
                      
                      
                         
                      
                      
                        
                          
                            
                              
                              
                              
                                Playlists
                                Build personalized lists of content—from your favorites to topic–centric collections or this weekend's reading list—and return to them whenever you have time.
                                
                                  
                                    Discover playlists
                                    
                                  
                                
                              
                            
                          
                        
                      
                      
                          
                      
                      
                         
                      
                      
                
                  
                    
                      
                        
                          
                            
                              
                                There's more where this came from.
                                Stay ahead of the curve when you purchase a month–to–month membership for just $39 per month or a yearlong membership for just $399 (a 15% savings).
                                
                                  
                                    Become a member
                                    
                                  
                                
                              
                            
                          
                        
                      
                    
                  
                

                
                
                
                
                   
                
              
              
              
                
                   
                
                
                
                  
                    
                      
                        
                        
                        
                        
                        
                        
                        
                        
                        
                        
                      
                    
                  
                
                
                   
                
                
                
                  
                    This message was sent to reply@reply.github.com. You are receiving this because you're an O'Reilly online learning member, or you've signed up to receive email from us. We hope you found this message to be useful. However, if you'd rather not receive future emails of this type from O'Reilly, please manage your preferences or unsubscribe here.
                    
                    Read our Privacy Policy.
                    
                    O'Reilly Media, Inc., 1005 Gravenstein Highway North, Sebastopol, CA, 95472 (707) 827-7000

**Comment by PaulWestphal** [comment]
fixed ff8ead3bb9e94558a586f2bc4c0acff5fb5c48d2
